go/logic/inspect.go

```diff
@@ -195,12 +196,22 @@ func (this *Inspector) validateConnection() error {
 	if len(this.connectionConfig.Password) > mysql.MaxReplicationPasswordLength {
 		return fmt.Errorf("MySQL replication length limited to 32 characters. See https://dev.mysql.com/doc/refman/5.7/en/assigning-passwords.html")
 	}
-	query := `select @@global.port, @@global.version`
+
+	extraPortRegexp := regexp.MustCompile("Percona|Maria")
+	query := `select @@global.port, @@global.version, @@global.version_comment`
 	var port int
-	if err := this.db.QueryRow(query).Scan(&port, &this.migrationContext.InspectorMySQLVersion); err != nil {
+	var extraPort int
+	var versionComment string
+	if err := this.db.QueryRow(query).Scan(&port, &this.migrationContext.InspectorMySQLVersion, &versionComment); err != nil {
```

Comment:
This bit could be DRY'd up, but I wanted to get something in front of ya'll first. I can iterate to clean up some of this if you all want.

Reply:
yes please :)